app/external/CMakeLists.txt

```diff
@@ -17,8 +32,9 @@ project(AubioBuilder
 ExternalProject_Add(sp_midi
     PREFIX sp_midi-prefix
     SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/sp_midi
-    INSTALL_COMMAND ""
+    INSTALL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../server/beam/tau/priv
```

Comment:
So right here, because sp_midi and sp_link are built _after_ `mix release` has been called (in `linux-pre-tau-prod-release.sh`), the libraries don't get copied into the distributed BEAM server files under `server/beam/tau/_build/prod/rel/tau/lib/tau-0.1.0/priv` and lead to the behavior @rbnpi mentioned in #3129 that I also experienced

These files need to be copied before `mix release` is run so that might have to come at the end of the build process (i.e. post-CMake) for this to work right